packages/backend/migration_status.txt

```diff
@@ -0,0 +1,4 @@
+20241004010448/installed document refs
+20241025030906/installed users
+20250114010822/installed test first five view
+20250121193606/installed testing
```

Comment:
What is this file for? I can't find any reference to it.

It's somewhat concerning to be tracking the status of migrations outside of the db, the authoritative source for migration status is the `_sqlx_migrations` table. Let me know if you need any help with migration stuff.